shopinvader_search_engine/models/se_index.py

```diff
@@ -11,11 +11,14 @@ class SeIndex(models.Model):
     is_valid = fields.Char(compute="_compute_is_valid")
 
     @api.depends("lang_id", "model_id")
+    @api.depends_context("shopinvader_backend_id")
     def _compute_is_valid(self):
+        backend = self.env["shopinvader.backend"]
+        backend_id = self.env.context.get("shopinvader_backend_id")
+        if backend_id:
+            backend = backend.browse(backend_id)
         for rec in self:
-            active_id = self.env.context.get("shopinvader_backend_id")
-            active_id = self.env["shopinvader.backend"].browse(active_id)
-            if active_id and rec.lang_id in active_id.lang_ids:
+            if not rec.lang_id or not backend:
                 rec.is_valid = True
```

Comment:
rec.is_valid when no backend ?

Reply:
sure. If we are not in the context of a backend the index is always valid (See `depends_context`)